src/guide/1/pubsub/javascript/snippets/subscribe.js

```diff
@@ -0,0 +1,17 @@
+// Creates a filter that defines that the 'message' field exists
+const filter = {exists: {field: 'message'}};
+// Will be triggered each time a document matches the filter
+const callback = (notification) => {
```

Comment:
(optional / nitpicking)

```suggestion
const callback = notification => {
```